code/modules/research/nanites/nanite_programs/protocols.dm

```diff
@@ -46,43 +46,25 @@
 	factory_efficiency = min(factory_efficiency + 1, max_efficiency)
 	nanites.adjust_nanites(null, round(0.002 * factory_efficiency, 0.1))
 
-/datum/nanite_program/protocol/tinker
-	name = "Tinker Protocol"
-	desc = "Replication Protocol: the nanites learn to use metallic material in the host's bloodstream to speed up the replication process."
+/datum/nanite_program/protocol/pyramid
+	name = "Pyramid Protocol"
+	desc = "Replication Protocol: the nanites implement an alternate cooperative replication protocol that is more efficient as long as the saturation level is above 80%."
 	use_rate = 0
 	rogue_types = list(/datum/nanite_program/necrotic)
 	protocol_class = NANITE_PROTOCOL_REPLICATION
-	var/boost = 2
-	var/list/valid_reagents = list(
-		/datum/reagent/iron,
-		/datum/reagent/copper,
-		/datum/reagent/gold,
-		/datum/reagent/silver,
-		/datum/reagent/mercury,
-		/datum/reagent/aluminium,
-		/datum/reagent/silicon)
-
-/datum/nanite_program/protocol/tinker/check_conditions()
-	if(!nanites.host_mob.reagents)
-		return FALSE
-
-	var/found_reagent = FALSE
+	var/boost = 1.2
 
-	var/datum/reagents/R = nanites.host_mob.reagents
-	for(var/VR in valid_reagents)
-		if(R.has_reagent(VR, 0.5))
-			R.remove_reagent(VR, 0.5)
-			found_reagent = TRUE
-			break
-	if(!found_reagent)
+/datum/nanite_program/protocol/pyramid/check_conditions()
+	if((nanites.nanite_volume / nanites.max_nanites) < 0.8)
```

Comment:
scales on nanite volume when you add a protocol that gives extra regen but gives you less nanite volume, not poggers